timed/projects/views.py

```diff
@@ -171,20 +189,44 @@ class TaskAsssigneeViewSet(ReadOnlyModelViewSet):
     filterset_class = filters.TaskAssigneeFilterSet
 
     def get_queryset(self):
-        return models.TaskAssignee.objects.select_related("task", "user")
+        """Don't show task assignees to customers."""
+        user = self.request.user
+
+        queryset = models.TaskAssignee.objects.select_related("task", "user")
+
+        if models.CustomerAssignee.objects.filter(user=user, is_customer=True).exists():
```

Comment:
Deny access by default, only allow if explicitly needed. This code only forbids to customers, but everyone else sees all the data